json/fr.json

```diff
@@ -77,10 +77,10 @@
     "Leave Team" : "Quitter l'équipe",
     "Leave" : "Quitter",
     "Login": "Connexion",
-    "Logout Other Browser Sessions" : "Déconnecter les sessions ouvertes sur d'autres navigateurs",
-    "Logout": "Déconnexion",
```

Comment:
Keys used in Laravel products cannot be deleted.

Reply:
Although 'Login' and 'Logout' were superseded in the latest versions of all Laravel products

Reply:
Oh... https://github.com/laravel/jetstream/pull/677/files?file-filters%5B%5D=.php